pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -52,6 +59,10 @@ pub mod pallet {
 	pub(crate) type StateMigrationStatusValue<T: Config> =
 		StorageValue<_, (StateMigrationStatus, u64), ValueQuery>;
 
+	#[pallet::storage]
+	pub(crate) type ForeignAssetMigrationStatusValue<T: Config> =
+		StorageValue<_, ForeignAssetMigrationStatus, ValueQuery>;
```

Comment:
A simple value for the whole migration imply that we can migrate only once asset at a time.
IIRC, the foundation wanted to be able to migrate several assets at the same time

Reply:
I introduced a change that allows approving the migration of multiple assets in one call. Once approved, the assets can then be migrated one by one. This approach ensures that each asset is migrated as quickly as possible after the migration begins, minimizing the time it remains frozen during migration.